modules/liveIntentIdSystem.js

```diff
@@ -64,18 +82,30 @@ function initializeLiveConnect(configParams) {
   if (configParams.partner) {
     identityResolutionConfig.source = configParams.partner
   }
+  if (configParams.ajaxTimeout) {
```

Comment:
there is one more param left that we don't pass: `expirationDays`. Should we or can we just keep the default 1 day?

Reply:
Hm.... i think we should set it to 0 days, otherwise prebid controlls for cookie expiry don't make any sense.

Only thing prebid would do if the cookie retention is 6 hours would be to copy over our cookie value with the one from pbjs.

Reply:
this was changed to be 1 hour by default